manager/src/main/java/com/redhat/service/smartevents/manager/ErrorHandlerServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package com.redhat.service.smartevents.manager;
+
+import java.util.Map;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+
+import com.redhat.service.smartevents.infra.models.gateways.Action;
+import com.redhat.service.smartevents.manager.providers.InternalKafkaConfigurationProvider;
+import com.redhat.service.smartevents.manager.providers.ResourceNamesProvider;
+
+@ApplicationScoped
+public class ErrorHandlerServiceImpl implements ErrorHandlerService {
+
+    @Inject
+    InternalKafkaConfigurationProvider internalKafkaConfigurationProvider;
+    @Inject
+    ResourceNamesProvider resourceNamesProvider;
+
+    @Override
+    public Action getDefaultErrorHandlerAction() {
```

Comment:
@kostola Perhaps just `getAction()` as the method name? I find the `getDefaultErrorHandler` prefix a bit misleading as we don't apply this by default. A customer still has to actively decide to enable this error handler.